### ISSUE TITLE
Add prettier to react-kit

### ISSUE THREAD

**Issue by sutarmin** [devex-web-frontend/dx-platform#8]
Add [prettier](https://www.npmjs.com/package/prettier) to react-kit package. 
I suggest following `.prettierrc` (from one of our projects, may be discussed):
```
{
  "bracketSpacing": true,
  "jsxBracketSameLine": true,
  "parser": "typescript",
  "printWidth": 120,
  "semi": true,
  "singleQuote": true,
  "tabWidth": 4,
  "trailingComma": "all",
  "useTabs": true
}
```

This task includes following steps in the root of the repo:
1) `npm i prettier` 
2) adding `.prettierrc` to the root folder (content above)
3) adding scripts `npm run prettier` and `npm run prettier:fix` to `package.json`
4) run `npm run prettier:fix` . Please commit the result separately. Same branch, but separate commit. This will make review easier.

**Comment by raveclassic** [comment]
should be the same as in our current closed-source project

**Comment by sutarmin** [comment]
This task includes following steps in the react-kit package:
1) `npm i prettier` 
2) `.prettierrc` in react-kit root folder (content above)
3) adding scripts `npm run prettier` and `npm run prettier:fix` to `package.json`
4) run `npm run prettier:fix` . Please commit the result separately. Same branch, but separate commit. This will make review easier.

**Comment by scink** [comment]
how about make it for the whole platform, not only for the kit?

**Comment by scink** [comment]
discussed it with @raveclassic, let's add .prettierrc to the root directory so prettier could check all packages of the platform

**Comment by sutarmin** [comment]
fixed description due to above discussion

**Comment by sutarmin** [comment]
fixed by #17
